tools/wt_binary_decode.py

```diff
@@ -646,17 +665,19 @@ def wtdecode(opts):
 import argparse
 parser = argparse.ArgumentParser()
 parser.add_argument('-b', '--bytes', help="show bytes alongside decoding", action='store_true')
+parser.add_argument('--bson', help="decode cell values as bson data", action='store_true')
+parser.add_argument("-c", "--csv", action='store', type=argparse.FileType('w'), dest='output', help="Directs the output to a name of your choice")
```

Comment:
I know this PR didn't add this code, but since it's being moved/modified, we should clean this up this line.  Can we say:
```
help="output filename for summary of page statistics in CSV format"
```
and put `action='store'` at the end.

Reply:
Sounds good